utils.py

```diff
@@ -88,8 +89,12 @@ def compute_ram_memory_stats(byte_scaler =1024 ** 3):
 
 
 def compute_memory_fragmentation():
-	snapshot = torch.cuda.memory_snapshot()
```

Comment:
why did you remove the memory fragmentation code? it is still useful

Reply:
Because there is only one use of this function. Old function cause division by zero. The new one works for both cases onnx and pytorch.

Reply:
1. please fix division by zero
2. the semantics of the old and new functions is completely different, so it does not make sense to change the implementation of the old one
3. feel free to add measurements with the new function

Reply:
the old function measures fragmentation, the new function measures utilisation. these are not the same thing

Reply:
function semantic is changed